server/handlers/converters.go

```diff
@@ -38,6 +38,9 @@ func (ah *ActivityHandler) msgToPost(channelID, senderID string, msg *msteams.Me
 	var embeddedImages []msteams.Attachment
 	text, embeddedImages = ah.handleImages(text)
 	msg.Attachments = append(msg.Attachments, embeddedImages...)
+	if !strings.HasPrefix(text, "<p>") {
```

Comment:
I think a better way to do it to put it as an exception in the `ConvertToMD`  function, that has a check at the beginning for text that contains `<div>` or  `<p>`, you can add `<img ` or any other chunk of text that clearly means that there is some HTML to be process.